public/New-DbaDbUser.ps1

```diff
@@ -88,136 +91,132 @@ function New-DbaDbUser {
 
         Creates a new sql user named 'claudio@********.onmicrosoft.com' mapped to Azure Active Directory (AAD) in the specified database.
 
+    .EXAMPLE
+        PS C:\> New-DbaDbUser -SqlInstance sqlserver2014 -Database DB1 -Username user1 -Password (ConvertTo-SecureString -String "DBATools" -AsPlainText)
+
+        Creates a new cointained sql user named user1 in the database given database with the password specified.
     #>
     [CmdletBinding(SupportsShouldProcess, ConfirmImpact = "Medium")]
     param(
-        [parameter(Mandatory, Position = 1)]
+        [parameter(Mandatory = $True, Position = 1)]
         [DbaInstanceParameter[]]$SqlInstance,
         [PSCredential]$SqlCredential,
         [string[]]$Database,
         [string[]]$ExcludeDatabase,
-        [switch]$IncludeSystem,
+        [switch]$IncludeSystem = $False,
+        [Parameter(ParameterSetName = "Login", Mandatory = $True)]
         [string]$Login,
-        [string]$Username = $Login,
+        [Parameter(ParameterSetName = "Login", Mandatory = $False)]
+        [Parameter(ParameterSetName = "NoLogin", Mandatory = $True)]
```

Comment:
```suggestion
        [Parameter(Mandatory, ParameterSetName = "NoLogin")]
```

Reply:
I prefer to keep it declared so that it is easy for any newbie like me to understand it clearly